sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -162,6 +162,28 @@ static pi_result USMSetIndirectAccess(pi_kernel kernel) {
 
 extern "C" {
 
+pi_result piDeviceGetInfo(pi_device device, pi_device_info paramName,
+                          size_t paramValueSize, void *paramValue,
+                          size_t *paramValueSizeRet) {
+  switch (paramName) {
+    // Intel GPU EU device-specific information extensions
```

Comment:
I thought some of support is coming in OpenCL RT. Can you maybe put a link here so that we can support it where that is landed?

Reply:
@bashbaug wrote a proposal to standardize PCI BDF query.  
[https://gitlab.khronos.org/opencl/OpenCL-Docs/-/issues/244]
[https://gitlab.khronos.org/opencl/OpenCL-Docs/-/merge_requests/171]
The device descriptor described there will return a structure.   My PCI BDF device descriptor returns a string (because this is what was requested).

I only looked for PCI address support.  Once I was aware that this support would be limited to Level Zero, I did not look further.

Reply:
Sorry - what are you asking for Sergey?  A link to the compute-runtime github that implements OpenCL on Intel GPUs?
A link to the OpenCL headers if a new enum value appears?

Reply:
I wanted a link that (if clicked) would indicate that OpenCL RT got this extension merged (meaning we could add the support to SYCL).

Reply:
I do not know what that link would be...

Reply:
OK, then just mention please in a TODO comment that this is being added and we should check back.